docker/AccountManager.Dockerfile

```diff
@@ -0,0 +1,13 @@
+# syntax=docker/dockerfile:1
+FROM python:3.9.9-buster
+
+WORKDIR /usr/local/share/
+RUN git clone https://github.com/DarkflameUniverse/AccountManager
```

Comment:
Thoughts on whether this dockerfile should be here vs in the AccountManager repo (where it already exists) or both?

Reply:
Good thought. You could add that repo as a git submodule and just point the docker-compose file at it

Reply:
I agree, I moved it to submodules.